src/sentry/sentry_metrics/configuration.py

```diff
@@ -0,0 +1,61 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import MutableMapping, Optional
+
+from django.conf import settings
+
+
+class ProfileKey(Enum):
+    RELEASE_HEALTH = "release-health"
+    PERFORMANCE = "performance"
+
+
+DEFAULT_PROFILE_KEY = ProfileKey.RELEASE_HEALTH
+
+
+@dataclass(frozen=True)
+class MetricsIngestConfiguration:
+    db_model: ProfileKey
+    input_topic: str
+    output_topic: str
+    use_case_id: str
+    internal_metrics_tag: Optional[str]
+
+
+_METRICS_INGEST_CONFIG_BY_PROFILE_KEY: MutableMapping[
+    ProfileKey, MetricsIngestConfiguration
+] = dict()
+_METRICS_INGEST_CONFIG_BY_USE_CASE: MutableMapping[str, MetricsIngestConfiguration] = dict()
```

Comment:
Why do we need both `ProfileKey` and `use_case_id` ?
Can't the `ProfileKey` be the way to identify a use case id, and the `db_model` attribute just reference the model class name ?

Reply:
yeah, that's a good point. I think that's the result of a series of local refactors that in combination don't make a lot of sense